pyproject.toml

```diff
@@ -8,7 +8,7 @@ readme = "README.md"
 homepage = "https://github.com/vocodedev/vocode-python"
 
 [tool.poetry.dependencies]
-python = ">=3.8.1,<3.12"
+python = ">=3.9,<4.0"
```

Comment:
I believe it is reasonable to deprecate Python 3.8 support. However, setting the upper limit to <4.0 might be too aggressive. Let's take a more conservative approach

Reply:
what do you think of this @ajar98 ?

Reply:
Personally I like the approach of not setting upper bounds until you have found a new version that is actually incompatible. Setting an upper restriction on the python version here might mean in future that a user is forced to wait for a vocode update in order to use a new version of python / new packages in their project. Reasoning / related thinking is described in https://iscinumpy.dev/post/bound-version-constraints/